src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -257,6 +257,16 @@ public enum NuGetLogCode
         /// </summary>
         NU3023 = 3023,
 
+        /// <summary>
+        /// Timestamp certificate chain building failed
+        /// </summary>
+        NU3041 = 3041,
+
+        /// <summary>
+        /// Timestamp certificate not yet effective
+        /// </summary>
+        NU3044 = 3044,
```

Comment:
should this be `NU3032`?

Reply:
as discussed offline, this is based off of Damon's spec on errors. Pato is adding the rest in separate PR.